applications/tari_console_wallet/src/ui/state/wallet_event_monitor.rs

```diff
@@ -79,22 +85,36 @@ impl WalletEventMonitor {
                         match result {
                             Ok(msg) => {
                                 trace!(target: LOG_TARGET, "Wallet Event Monitor received wallet event {:?}", msg);
-                                     match (*msg).clone() {
-                                        ConnectivityEvent::PeerDisconnected(_) |
-                                        ConnectivityEvent::ManagedPeerDisconnected(_) |
-                                        ConnectivityEvent::PeerConnected(_) |
-                                        ConnectivityEvent::PeerBanned(_) |
-                                        ConnectivityEvent::PeerOffline(_) |
-                                        ConnectivityEvent::PeerConnectionWillClose(_, _) => {
-                                            self.trigger_peer_state_refresh().await;
-                                        },
-                                        /// Only the above variants trigger state refresh
-                                        _ => (),
-                                     }
+                                match (*msg).clone() {
+                                    ConnectivityEvent::PeerDisconnected(_) |
+                                    ConnectivityEvent::ManagedPeerDisconnected(_) |
+                                    ConnectivityEvent::PeerConnected(_) |
+                                    ConnectivityEvent::PeerBanned(_) |
+                                    ConnectivityEvent::PeerOffline(_) |
+                                    ConnectivityEvent::PeerConnectionWillClose(_, _) => {
+                                    self.trigger_peer_state_refresh().await;
+                                    },
+                                    /// Only the above variants trigger state refresh
+                                    _ => (),
+                                }
                             },
                             Err(_) => debug!(target: LOG_TARGET, "Lagging read on Connectivity event broadcast channel"),
                         }
                     },
+                    result = base_node_events.select_next_some() => {
+                        match result {
+                            Ok(msg) => {
+                                trace!(target: LOG_TARGET, "Wallet Event Monitor received base node event {:?}", msg);
+                                match (*msg).clone() {
+                                    BaseNodeEvent::BaseNodeState(state) => {
+                                        debug!(target: LOG_TARGET, "base node state: {:?}", state);
```

Comment:
I don't think this log is valuable, you do log it as a trace a couple lines above and I would argue these should be trace anyway so I would drop this one.